craft_application/services/testing.py

```diff
@@ -48,17 +50,26 @@ def process_spread_yaml(self) -> None:
             craft_backend=craft_backend,
         )
 
-        spread_yaml.to_yaml_file(pathlib.Path("spread/spread.yaml"))
+        spread_yaml.to_yaml_file(dest)
+
+    def run_spread(self, project_path: pathlib.Path) -> None:
+        """Run spread on the processed project file.
+
+        :param project_path: The processed project file.
+        """
+        with emit.pause():
+            os.environ["SPREAD_PROJECT_FILE"] = str(project_path)
+            subprocess.run(["spread", "-v", "craft:"], check=True)
 
     def _get_backend(self) -> models.SpreadBackend:
-        name = "lxd-vm"
+        name = "ci" if os.environ.get("CI") else "lxd-vm"
```

Comment:
Should it be handled by the `ConfigService` ?
I think it at least it should be `CRAFT_TEST_CI` to avoid the collision and indicate that it is for the consume of the craft aplication

Reply:
`CI` is the "industry-standard" variable set by GitHub CI and others. We use it to verify if we're running on a CI environment or not.